pstl/include/pstl/internal/execution_impl.h

```diff
@@ -95,16 +95,14 @@ using __allow_parallel =
     typename __internal::__policy_traits<typename std::decay<_ExecutionPolicy>::type>::__allow_parallel;
 
 template <typename _ExecutionPolicy, typename... _IteratorTypes>
-typename std::conjunction<__allow_vector<_ExecutionPolicy>,
-                          __is_random_access_iterator<_IteratorTypes>...>::type
+typename std::conjunction<__allow_vector<_ExecutionPolicy>, __is_random_access_iterator<_IteratorTypes>...>::type
```

Comment:
Those changes seem unrelated to this commit. Likely it's clang-format applying.

Reply:
yes.. I'll redo that.